web/packages/teleport/src/services/api/api.test.ts

```diff
@@ -41,6 +41,7 @@ describe('api.fetch', () => {
     // Override the default header from `defaultRequestOptions`.
     headers: {
       Accept: 'application/json',
+      'Content-Type': 'application/json; charset=utf-8',
```

Comment:
this will be problematic, see [here](https://github.com/gravitational/teleport/blob/d4e629da38d4cbceb2033e625ce768182c426513/web/packages/teleport/src/services/api/api.ts#L53)

you can test this out by picking a integration to create. go to `enroll new integration` page and you can add whatever values to inputs and try to save, backend won't be able to read form data values

Reply:
Ok i'll remove and just add it to my callsite